tests/src/test/java/alluxio/master/file/FileSystemMasterClientRestApiTest.java

```diff
@@ -141,7 +142,9 @@ public void getStatus() throws Exception {
     Assert.assertEquals(0, fileInfo.getLength());
   }
 
+  // TODO(binfan): re-enable this test
   @Test
+  @Ignore
```

Comment:
agreed, re-enabling this test is low priority in my mind as it is testing API that is deprecated